pages/ksphere/kommander/1.1/clusters/attach-cluster/federated-addons/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: layout.pug
+navigationTitle: Federated Addons
```

Comment:
We're not using the terms "federated" and "addon" on the other docs IIRC. Maybe we need to explain them in one of the pages further up the page hierarchy.

Reply:
Now introduced in the attach-cluster `index.md`.